app/models/exchange_rate_collection.rb

```diff
@@ -5,7 +5,14 @@ class ExchangeRateCollection
 
   set_singular_path '/exchange_rates/:id'
 
-  attr_accessor :month, :year
+  attr_accessor :month, :year, :type
+
+  enum :type, {
+    monthly: %w[scheduled],
```

Comment:
As you know this sounds a bit strange,
since scheduled also means Spot.
But that fix is probably for another occasion.